src/service/recruit/posting.ts

```diff
@@ -1,11 +1,18 @@
-import { InputState, Keyword } from '../../types';
+import { EditPosting, InputState, Keyword } from '../../types';
 import { axiosAuthInstance, axiosInstance } from '..';
 import { EndPoint } from '..';
 
+interface Result {
+	recruitmentPostId: number;
+}
+
 export const postingRecruit = async (formData: InputState) => {
 	try {
-		const id = await axiosAuthInstance.post(EndPoint.RECRUITMENT.post, formData);
-		return id;
+		const response: Result | undefined = await axiosAuthInstance.post(
+			EndPoint.RECRUITMENT.post,
+			formData
+		);
+		return response;
 	} catch (error) {
 		console.error(error);
 	}
```

Comment:
```suggestion
export const postingRecruit = async (formData: InputState) => {
	try {
		const response = await axiosAuthInstance.post<Result | undefined>(
			EndPoint.RECRUITMENT.post,
			formData
		);
		return response;
	} catch (error) {
		console.error(error);
	}
```

그리고 undefined 타입을 추가하신 이유를 알 수 있을까용??

Reply:
undefined를 반환할 수 있어서 타입 에러가 났었는데, 그 부분을 급하게 해결하기 위해서 저렇게 작성했습니다.

말씀해주셔서 감사한게 이 부분은 디테일하게 신경쓰지 못했었습니다...
그래서 현재 undefined가 되어도 생성이 되었는데, 
해당 부분을 수정해서 이제 undefined라면 에러 발생시키고 구인글 생성이 되지 않습니다! 
그리고 타입 에러도 발생하지 않구요!

<img width="762" alt="image" src="https://github.com/MeeTeamNumdle/MeeTeam_FrontEnd/assets/76275691/afc03dc4-4d51-4460-b154-40ea140e59c2">